bitnami/mysql/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: mysql
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/mysql
-version: 9.12.1
+version: 9.12.2
```

Comment:
Please address the warning and ensure you add a newline character at the end of file.

Reply:
Added the extra line post version in Chart.yaml